tinygrad/tensor.py

```diff
@@ -2688,6 +2688,19 @@ def celu(self, alpha=1.0):
     """
     return self.maximum(0) + (alpha * ((self / alpha).exp() - 1)).minimum(0)
 
+  def selu(self, alpha=1.67326, scale=1.0507):
+    """
+    Applies the Scaled Exponential Linear Unit (SELU) function element-wise.
+
+    - Described: https://paperswithcode.com/method/selu
+    - Paper: https://arxiv.org/abs/1706.02515v5
+
+    ```python exec="true" source="above" session="tensor" result="python"
+    print(Tensor([-3., -2., -1., 0., 1., 2., 3.]).softsign().numpy())
+    ```
+    """
+    return scale * (self.relu() - (-alpha * self.exp() + alpha).relu())
```

Comment:
`return scale * (self > 0).detach().where(self, alpha * (self.exp() - 1))`
which implementation is cleaner? ^ This one is slightly faster and looks more like the implementation in `https://paperswithcode.com/method/selu`

Reply:
yea i think `scale * (self >= 0).detach().where(self, alpha * (self.exp() - 1))` is more readable